doc/tutorials/tarball_installation.rst

```diff
@@ -0,0 +1,248 @@
+  has an Ubuntu OS, ACRN dependencies, and ACRN hypervisor and kernel source
+  code).
+
+Build the Tarball Files
+***************************
+
+#. On your development computer, navigate to the ``acrn-hypervisor`` source code
+   and build the ACRN hypervisor tarball files. Be sure to replace
+   ``<board.xml>`` and ``<scenario.xml>`` with the paths to your board
+   configuration file and scenario configuration file.
+
+   .. code-block:: bash
+
+      cd ~/acrn-work/acrn-hypervisor
+      make -j $(nproc) BOARD=<board.xml> SCENARIO=<scenario.xml>
+      make targz-pkg
+
+#. Navigate to the ``acrn-kernel`` source code and build the ACRN kernel tarball
+   files for the Service VM:
+
+   .. code-block:: bash
+
+      cd ~/acrn-work/acrn-kernel
+      cp kernel_config_service_vm .config
+      make olddefconfig
+      make -j $(nproc) targz-pkg
+
+#. Copy all the necessary files generated on the development computer to the
+   target system by USB disk as follows. Modify the file names in the following
+   commands to match your files.
+
+   a. Insert the USB disk into the development computer and run these commands:
+
+      .. code-block:: bash
+
+         disk="/media/$USER/"$(ls /media/$USER)
+         cp linux-5.10.65-acrn-service-vm-x86.tar.gz "$disk"/
+         cp ~/acrn-work/acrn-hypervisor/build/hypervisor/acrn.bin "$disk"/
+         cp ~/acrn-work/acpica-unix-20210105/generate/unix/bin/iasl "$disk"/
+         cp ~/acrn-work/acrn-hypervisor/build/acrn-2.7-unstable.tar.gz "$disk"/
+         sync && sudo umount "$disk"/
+
+      .. note:: The :file:`serial.conf` is only generated if non-standard
+         vUARTs (not COM1-COM4)
+         are configured for the Service VM in the scenario XML file.
+         Please copy the ``serial.conf`` file using::
+            
+            cp ~/acrn-work/acrn-hypervisor/build/hypervisor/serial.conf "$disk"/
+
+   #. Insert the USB disk you just used into the target system and run these
+      commands to copy the tar files locally:
+
+      .. code-block:: bash
+
+         disk="/media/$USER/"$(ls /media/$USER)
+         cp "$disk"/linux-5.10.65-acrn-service-vm-x86.tar.gz ~/acrn-work
+         cp "$disk"/acrn-2.7-unstable.tar.gz ~/acrn-work
+
+   #. Extract the Service VM files onto the target system:
+
+      .. code-block:: bash
+
+         cd ~/acrn-work
+         sudo tar -zxvf linux-5.10.65-acrn-service-vm-x86.tar.gz -C / --keep-directory-symlink
+
+   #. Extract the ACRN tools and images:
+
+      .. code-block:: bash
+
+         sudo tar -zxvf acrn-2.7-unstable.tar.gz -C / --keep-directory-symlink
+
+   #. Copy a few additional ACRN files to the expected locations:
+
+      .. code-block:: bash
+
+         sudo mkdir -p /boot/acrn/
+         sudo cp "$disk"/acrn.bin /boot/acrn
+         sudo cp "$disk"/serial.conf /etc
+         sudo cp "$disk"/iasl /usr/sbin/
+         sync && sudo umount "$disk"/
+
+Install the Tarball Files
```

Comment:
This chapter really starts when you untar the files on the target device rootfs, so I would move the section title to just before the "Insert the USB disk you just used into the target..." step.

Reply:
This section could still be a different section but I would then call it something like: "Final Configuration steps" or something like that.

Reply:
I placed the heading a little higher than your recommendation. Please review.